packages/components/src/FilterBar/_docs/FilterBar.stories.tsx

```diff
@@ -216,6 +216,73 @@ export const OnValuesChange: StoryFn<typeof FilterBar> = () => {
   )
 }
 
+export const DependentFilter: StoryFn<typeof FilterBar> = () => {
+  type ValuesDependent = {
+    flavour: string
+    topping: string
+    drank: Date
+  }
+
+  const filtersDependent = [
+    {
+      id: "flavour",
+      name: "Flavour",
+      Component: (
+        <FilterBar.Select
+          items={[
+            { value: "jasmine-milk-tea", label: "Jasmine Milk Tea" },
+            { value: "honey-milk-tea", label: "Honey Milk Tea" },
+            { value: "lychee-green-tea", label: "Lychee Green Tea" },
+          ]}
+        />
+      ),
+    },
+    {
+      id: "topping",
+      name: "Topping",
+      Component: (
+        <FilterBar.Select
+          items={[
+            { value: "pearls", label: "Pearls" },
+            { value: "fruit-jelly", label: "Fruit Jelly" },
+          ]}
+        />
+      ),
+      isUsableWhen: state => state.flavour.value !== undefined,
+    },
+    {
+      id: "drank",
+      name: "Drank",
+      Component: <FilterBar.DatePicker />,
+      isRemovable: true,
+      isUsableWhen: state => state.flavour.value !== undefined,
```

Comment:
Just an idea here after trying out the Story:
What if we made the `Drank` filter dependent on `Topping`? Would make the example a bit more elaborate and show you can make something dependent on a filter that is dependent on something else?